TestMatrix.md

```diff
@@ -22,44 +22,48 @@ The module works against a broad range of IOS based devices, but we don't test a
+| radius_server_group            | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| search_domain                  | use network_dns      | use network_dns      | use network_dns      | use network_dns      | use network_dns      | use network_dns      | use network_dns      |
+| snmp_community                 | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| snmp_notification              | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| snmp_notification_receiver     | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| snmp_user*                     | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| syslog_server                  | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| syslog_settings                | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| tacacs                         | not supported by IOS | not supported by IOS | not supported by IOS | not supported by IOS | not supported by IOS | not supported by IOS | not supported by IOS |
+| tacacs_global*                 | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| tacacs_server                  | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
+| tacacs_server_group            | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   | ok                   |
 
 Cells marked with the * have deviations. See the section below for details.
 
 ## Deviations
 
+### snmp_user
+
+As required by RFC 3414 SNMP server user commands will not be displayed in the configuration output of the device, therefore the likes of `password`, and `enforce_privacy` are not comparable on the device. This means when using the resource to manage v3 SNMP users then we cannot support idempotency and a corrective change will occur when `password` and `enforce_privacy` are present in a manifest.
```

Comment:
```suggestion
As required by RFC 3414 SNMP server, user commands will not be displayed in the configuration output of the device. Commands like `password` and `enforce_privacy` are not comparable on the device. This means when using the resource to manage v3 SNMP users, we can't support idempotency and a corrective change will occur when `password` and `enforce_privacy` are present in a manifest.
```

Reply:
Yes this suggestion works well, just changing it from 

```
Commands like `password` and `enforce_p.....
```

To something like:

```
The values `password` and `enforce_privacy` are not....
```

How does that sound? As these are values of the command vs the command itself.

Reply:
Yes sounds great!